static/app/components/events/autofix/autofixSolution.tsx

```diff
@@ -335,6 +332,8 @@ function getEventIcon(eventType: string) {
       return <IconCode {...iconProps} />;
     case 'human_instruction':
       return <IconUser {...iconProps} />;
+    case 'repro_test':
+      return <IconTimer {...iconProps} />;
```

Comment:
would an erlenmeyer flask icon be more associated with unit testing? 
![CleanShot 2025-03-14 at 09 40 06@2x](https://github.com/user-attachments/assets/e8e24372-c6fd-4168-ad25-9631e6dc1426)

Reply:
yk i've never seen that icon before
![Screenshot 2025-03-14 at 9 49 40 AM](https://github.com/user-attachments/assets/cc150b35-1a89-4217-8652-3ce0566181ff)

but I like it, I switched the icon.

I like the strikethrough though. I think it's a good hint that the other steps are editable too.

Reply:
lol orange react